### PR TITLE
Feature/docs(website) component page #50

### PR DESCRIPTION
**Pull request by anna-lach**
#50 

The documentation for every component should be kept in separated directory in `categories/components` directory, eg.:
**Button** should have path like `docs/website/src/site/categories/components/button`,
**Checkbox** should have path like `docs/website/src/site/categories/components/checkbox` etc. 

Every component directory consists of **7 files**, like for a `button` directory there will be:

- `accessibility.md` - file keeps content for the “**Accessibility**” tab for button component
- `button.json` - keeps information about layout and tags, layout should be the same for every component and `tags` should contain component name like `button` in this example
- `button.md` - `title` contains component name and `description` contains short component description
- `code.md` - file keeps content for the “**Code**” tab for button component
- `overview.md` - file keeps content for the “**Overview**” tab for button component
- `specs.md` - file keeps content for the “**Specs**” tab for button component
- `usage.md` - file keeps content for the “**Usage**” tab for button component

### CODE REVIEW

File: docs/website/src/site/_includes/components/components-overview.njk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: base.njk
+---
+
+<header class="ds-components__heading">
+    <h1 class="ds-heading-1">{{ title }}</h1>
```
Comment (jpzwarte) [inline docs/website/src/site/_includes/components/components-overview.njk:6]: Indentation looks wrong here? It's 4 spaces here, but 2 space 3 lines below?

File: docs/website/src/site/_includes/components/components.njk [hunk 2]
```diff
@@ -2,8 +2,59 @@
 layout: base.njk
 ---
 
-{#{% include "./tabs.njk" %}#} {# {% include "./overview.njk" %} {% include "./specs.njk" %}
-<h2>Usage</h2>
-<h2>Accessibility</h2>
-<h2>Code</h2>
-#} {{ content | safe }}
+  <header class="ds-component__heading">
```
Comment (jpzwarte) [inline docs/website/src/site/_includes/components/components.njk:5]: Indentation is wrong here as well?

Comment (jpzwarte) [reply docs/website/src/site/_includes/components/components.njk:5]: Looks to be wrong throughout the file?

File: docs/website/src/site/categories/components/button/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  parent: Button
+  key: ButtonAccessibility
+---
+
+<section> 
+
+## Accessibility
+
+Lorem ipsum dolor sit amet. Consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus. Maecenas eget condimentum velit, sit amet feugiat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Praesent auctor purus luctus enim egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex. Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in elementum tellus.
+
```
Comment (jpzwarte) [inline docs/website/src/site/categories/components/button/accessibility.md:14]: Are the newlines needed?

File: docs/website/src/site/categories/components/button/code.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+title: Button code
+tags: code
+eleventyNavigation:
+  parent: Button
+  key: ButtonCode
+---
+
+<section>
+
+  ## Code
```
Comment (jpzwarte) [inline docs/website/src/site/categories/components/button/code.md:11]: Identation is different in this file than it was in `accessibility.md`?

File: docs/website/src/site/scripts/tabs.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ for (let i = 0; i < tabs?.length; i++) {
     }
 
     const current = tabsContainer.querySelectorAll('.active');
```
Comment (jpzwarte) [inline docs/website/src/site/scripts/tabs.js:10]: Why do `querySelectorAll` if you're only going to use the first one? Simply do `querySelector`?

**Review by jpzwarte — APPROVED** [review]

File: docs/website/src/site/scripts/tabs.js (hunk 5 above)
Comment (anna-lach) [reply docs/website/src/site/scripts/tabs.js:10]: right :)

File: docs/website/src/site/categories/components/button/accessibility.md (hunk 3 above)
Comment (anna-lach) [reply docs/website/src/site/categories/components/button/accessibility.md:14]: not needed :) I'll remove them

File: docs/website/src/site/scripts/tabs.js (hunk 5 above)
Comment (anna-lach) [reply docs/website/src/site/scripts/tabs.js:10]: this is an old version of tabs, I forgot to remove it. The right file is in `ts/scripts/tabs.ts` file

**Pull request merged**